socorro/signature/rules.py

```diff
@@ -366,26 +367,37 @@ def generate(self, source_list, crashed_thread=None, delimiter=" | "):
         # have already been normalized at this point.
         new_signature_list = []
         for a_signature in source_list:
-            # If the signature matches the irrelevant signatures regex, skip to the next frame.
+            # If the frame signature is an empty string, skip it
+            if not a_signature:
+                debug_notes.append(f'empty string; ignoring: "{a_signature}"')
+                continue
```

Comment:
Previously, the frame signature could have at least a module offset, but we nixed that so now frame signatures for frames that are missing all the interesting information end up as `""`. In that case, we don't want it in the signature, so we continue to the next frame.

Reply:
Would it make sense to make the irrelevant signatures regex match the empty string instead?

Reply:
The irrelevant signatures regex is built from the `socorro/signature/siglists/irrelevant_signatures.txt` file. I don't have a good way to encode an empty string in that file.

So in this case, it seemed a lot easier to do this.